public/apps/account/test/account-app.test.tsx

```diff
@@ -57,6 +62,7 @@ describe('Account app', () => {
 
   beforeAll(() => {
     (fetchAccountInfoSafe as jest.Mock).mockResolvedValue(mockAccountInfo);
+    (fetchCurrentAuthType as jest.Mock).mockResolvedValue('dummy');
```

Comment:
`'dummy'` is not a valid auth type. How does the test work here?

Reply:
The currentAuthType is not being used until logout button triggers onClick function. So the return value for fetchCurrentAuthType is just a placeholder here.